scripts/commands/checkLinks.ts

```diff
@@ -295,4 +295,32 @@ async function determineHistoricalFileBatches(
   return result;
 }
 
+async function determineQiskitLegacyReleaseNotes(): Promise<FileBatch> {
+  const result: FileBatch[] = [];
+
+  const legacyVersions = (
+    await globby("docs/api/qiskit/release-notes/[!index]*")
+  )
+    .map((releaseNotesPath) =>
+      releaseNotesPath.split("/").pop()!.split(".").slice(0, -1).join("."),
+    )
+    .filter((version) => compareReleaseNotes(version, "0.45") < 0);
```

Comment:
We know there will never be another version between Qiskit 0.46 and 1.0. So, how about simplifying this to check if the version is `<1 && != 0.45 && != 0.46`? Good job writing generic code with `compareReleaseNotes`! But I think we can simplify this.

Note that the sorting doesn't matter either - it all gets checked in the same `FileBatch` either way.